README.md

```diff
@@ -15,7 +15,7 @@ Use your favourite plugin manager.
 
 Or use Vim8's built-in package support:
 ```sh
-git clone github.com/rottencandy/vimkubectl.git ~/.vim/pack/plugins/start/vimkubectl
+git clone git@github.com/rottencandy/vimkubectl.git ~/.vim/pack/plugins/start/vimkubectl
```

Comment:
Thanks! I didn't notice that the URL wasn't working.

But looks like it still needs the `:` for the ssh URL to work, do you mind changing it to
`git@github.com:rottencandy/vimkubectl.git`?

Reply:
Actually, I think changing it to HTTPS URL would be better, since it would not need a password:
`https://github.com/rottencandy/vimkubectl.git`